ecosystem/sep-0012.md

```diff
@@ -83,23 +88,25 @@ This endpoint allows clients to:
 ### Request
 
 ```
-GET [KYC_SERVER]/customer
-GET [KYC_SERVER]/customer?account=<public-key>&memo=<memo>&memo_type=<memo-type>&type=<customer-type>
-GET [KYC_SERVER]/customer?id=<customer-id>
+GET [KYC_SERVER]/customer&type=<customer-type>
+GET [KYC_SERVER]/customer?account=<stellar-account>&type=<customer-type>
+GET [KYC_SERVER]/customer?account=<stellar-account>&memo=<memo>&type=<customer-type>
+GET [KYC_SERVER]/customer?account=<muxed-account>&type=<customer-type>
+GET [KYC_SERVER]/customer?id=<customer-id>&type=<customer-type>
 ```
 
 Name | Type | Description
 -----|------|------------
 `id` | `string` | (optional) The ID of the customer as returned in the response of a previous `PUT` request. If the customer has not been registered, they do not yet have an `id`.
-`account` | `G...` string | (optional) The Stellar account ID used to identify this customer. If many customers share the same Stellar account, the `memo` and `memo_type` parameters should be included as well.
-`memo` | string | (optional) a [properly formatted memo](https://developers.stellar.org/docs/glossary/transactions/#memo) that uniquely identifies a customer. This value is generated by the client making the request. This parameter and `memo_type` are identical to the `PUT` request parameters of the same name.
-`memo_type` | string | (optional) type of `memo`. One of `text`, `id` or `hash`. If `hash`, `memo` should be base64-encoded.
+`account` | `G...` or `M...` string | (**deprecated**, optional) This field should match the `sub` value of the decoded SEP-10 JWT.
+`memo` | string | (optional) the client-generated [memo](https://developers.stellar.org/docs/glossary/transactions/#memo) that uniquely identifies the customer. If a memo is present in the decoded SEP-10 JWT's `sub` value, it must match this parameter value. If a muxed account is used as the JWT's `sub` value, requests containing a memo should be rejected. See the [Shared Accounts](#shared-omnibus-or-pooled-accounts) section for more information.
```

Comment:
I think its better to reject it, rather than assume the client made a mistake. If a muxed account is authenticated and a memo parameter is sent in the request, conceptually the client trying to create a sub account of a sub account, which isn't permitted.

Reply:
I generally agree that we shouldn't accept conflicting information, but in the interest of following [Postel's law](https://devopedia.org/postel-s-law) I think we could be flexible in the case where information is equivalent, i.e. when `memo == id_used_to_generate_muxed_address`. Our SDKs allow us to easily compare one with the other. What do you think?

Reply:
I can get behind that, I'll update the definition 👍